models/classes/menu/SectionVisibilityFilterInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\tao\model\menu;
+
+interface SectionVisibilityFilterInterface
+{
+    public function isHidden(string $section): bool;
```

Comment:
```suggestion
    public function isVisible(string $section): bool;
```
I wold use the same word `visible` to be consistent with the name `visibility`.

Reply:
Due to exclkuding strategy i would keep `isHidden` name. It's still in visibility domain (`isHidden`/`isVisible`)